### ISSUE TITLE
Gradient Accumulation Steps?

### ISSUE THREAD

**Issue by 2Raven2** [sorryhyun/anima_lora#33]
Title. Would this be something that can be added down the line as a way to increase batch sizes without raising VRAM cost? I'm not sure if `train_batch_size` even does anything; I set it to 4 but it still seems to still train at batch 1 but might not always be good in some cases as batch 2 or more is better to average the gradients during learning.

**Comment by sorryhyun** [comment]
hello, setting `gradient_accumulation_steps` in config (.toml) will help that, and you can increase batch size by
```
[general]
[[datasets]]
batch_size=2
```
in toml

**Comment by 2Raven2** [comment]
Would it be something that can be added in the GUI in the near future as an option? Thanks for the help!

**Comment by sorryhyun** [comment]
Sure I also agree that there are inconveniences.  Let me design that

**Comment by sorryhyun** [comment]
In v1.8.5, GUI knob in advanced settings was added.
